snippets/csharp/VS_Snippets_CLR_System/system.reflection.assembly.gettype/cs/gettype1.cs

```diff
@@ -4,39 +4,41 @@
 
 public class Example
 {
-   public static void Main()
-   {
-      Assembly assem = typeof(Example).Assembly;
-      Type t = assem.GetType("Transportation.MeansOfTransportation");
-      if (t != null) {
-         Console.WriteLine("Virtual properties in type {0}:", 
-                           t.FullName);
-         PropertyInfo[] props = t.GetProperties();
-         int nVirtual = 0;
-         for (int ctr = 0; ctr < props.Length; ctr++)
-            if (props[ctr].GetMethod.IsVirtual) {
-               Console.WriteLine("   {0} (type {1})",
-                                 props[ctr].Name, 
-                                 props[ctr].PropertyType.FullName);
-               nVirtual++;
-            }
+    public static void Main()
+    {
+        Assembly assem = typeof(Example).Assembly;
+        Type t = assem.GetType("Transportation.MeansOfTransportation");
+        if (t != null)
+        {
+            Console.WriteLine("Virtual properties in type {0}:",
```

Comment:
Consider switching to interpolated strings:

```suggestion
            Console.WriteLine($”Virtual properties in type {t.FullName}:");
```

Reply:
Thank you. I will switch it.